arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -0,0 +1,689 @@
+      "post": {
+        "tags": [
+          "Redis"
+        ],
+        "operationId": "Redis_ListKeys",
+        "description": "Retrieve a redis cache's access keys. This operation requires write permission to the cache resource.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the redis cache."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Lists the keys for Redis Cache",
+            "schema": {
+              "$ref": "#/definitions/RedisListKeysResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}/regenerateKey": {
+      "post": {
+        "tags": [
+          "Redis"
+        ],
+        "operationId": "Redis_RegenerateKey",
+        "description": "Regenerate redis cache's access keys. This operation requires write permission to the cache resource.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the redis cache."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RedisRegenerateKeyParameters"
+            },
+            "description": "Specifies which key to reset."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Lists the regenerated keys for Redis Cache",
+            "schema": {
+              "$ref": "#/definitions/RedisListKeysResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}/forceReboot": {
+      "post": {
+        "tags": [
+          "Redis"
+        ],
+        "operationId": "Redis_ForceReboot",
```

Comment:
Yes, the operation is really named forceReboot. AFAIK there were no complaints about this when the api was reviewed.